.travis

```diff
@@ -0,0 +1,23 @@
+language: python
+python:
+  - "2.7"
+addons:
+  postgresql: "9.4"
+before_install:
+  - sudo apt-get update
+  - pip install coverage
+  - pip install coveralls
+install:
+  - pip install -r requirements/dev.txt
+before_script:
+  - sudo createdb -U postgres openstax
+  - ./manage.py migrate --no-initial-data
+  - ./manage.py loaddata site.json
+#  - ./manage.py createsuperuser
+script:
+  - ./manage.py test
```

Comment:
Hold off on merging this branch even if tests pass because travis configuration because this command needs to be changed